portfolio/src/main/webapp/navbar.html

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
It looks like this file is not used in the final page. You should remove it from your branch (can always add it back later) to keep the repository "clean".